h2o-py/tests/testdir_algos/maxrglm/pyunit_PUBDEV_8235_maxrglm_gaussian_validation.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import print_function
+from __future__ import division
+import sys
+sys.path.insert(1, "../../../")
+import h2o
+from tests import pyunit_utils
+from h2o.estimators.maxrglm import H2OMaxRGLMEstimator as maxrglm
+
+# test maxrglm works with validation dataset
+def test_maxrglm_validation():
+    d = h2o.import_file(path=pyunit_utils.locate("smalldata/glm_test/gaussian_20cols_10000Rows.csv"))
+    myY = "C21"
```

Comment:
I prefer the Python syntax in the Python tests. It is a trifle, but can help make code much more readable. So it is up to you to change all variables to snake case, but I strongly recommend it. :)

```suggestion
    my_y = "C21"
```